<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_bg_of_login">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>350</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string>背景图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="lgl_title_of_app">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Goudy Old Style</family>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>富途资金管理平台</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_register">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>没有账号？去注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login_close">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_pwd_of_user">
    <property name="geometry">
     <rect>
      <x>461</x>
      <y>192</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_name_of_user">
    <property name="geometry">
     <rect>
      <x>461</x>
      <y>143</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入用户名</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>461</x>
      <y>252</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QRadioButton" name="chk_remember_me">
     <property name="text">
      <string>记住密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_confrim_login">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_login_warning">
    <property name="geometry">
     <rect>
      <x>463</x>
      <y>91</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edit_name_of_user</tabstop>
  <tabstop>edit_pwd_of_user</tabstop>
  <tabstop>btn_confrim_login</tabstop>
  <tabstop>btn_register</tabstop>
  <tabstop>chk_remember_me</tabstop>
  <tabstop>btn_login_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
